src/styles/main.scss

```diff
@@ -1,3 +1,28 @@
-body {
-  margin: 0;
+@import "reset";
+@import "header";
+@import "star";
+@import "card";
+@import "variables";
+
+html {
+  font-family: Roboto, serif;
+
+  .container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    flex-shrink: 0;
+    width: 1024px;
```

Comment:
[checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 11th item, use values like card-size, gap and max count of cards in a row to calculate it

Reply:
fix